src/current/_includes/releases/v23.1/v23.1.13.md

```diff
@@ -0,0 +1,88 @@
+## v23.1.13
+
+Release Date: December 5, 2023
+
+{{ site.data.alerts.callout_info }}
+This version is currently available only on select CockroachDB {{ site.data.products.dedicated }} clusters. If you wish to upgrade your cluster to this version, please submit a [support request](https://support.cockroachlabs.com/hc/en-us/requests/new).
+
+This release is currently not available for CockroachDB {{ site.data.products.serverless }} and {{ site.data.products.core }} customers.
+{{ site.data.alerts.end }}
```

Comment:
confirming that this language :lgtm: 
this also matches the language from the original admonition language we used for 23.1.11
https://docs.google.com/document/d/15Qkbg-WZx6JrEo5Qqft-k34PnVm_P_6tAAlkVH4FQBc/edit#heading=h.i6lmk2k77ao3

Reply:
awesome thanks!